recipes/fmjpeg2koj/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -eux # Abort on error.
+
+mkdir build
+cd build
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=${PREFIX} -DCMAKE_INSTALL_PREFIX=${PREFIX} -DFMJPEG2K=${PREFIX} ..
+
+cmake --build . --target install --parallel
```

Comment:
Is there a `make check` or `ctest` to run here?

Reply:
The source repo does not have any unit test. So, I just decided to check the existence of some headers and libraries once things are installed. Here is an example build script from the source repo: https://github.com/DraconPern/fmjpeg2koj/blob/master/build.sh

Does that answer your question?